documentation/staging/content/userguide/managing-operators/common-mistakes.md

```diff
@@ -0,0 +1,182 @@
+---
```

Comment:
A long title is a problem itself I think; hard to read.

Reply:
True, but not sure how to avoid it is this doc because the heading describes the mistake and having it written out in the TOC lets the user scan the headings to find their problem.

Reply:
I agree with Rosemary, and think the long titles are helpful in this particular case.

Reply:
We can leave this as it is for now and take care of it in a future PR, which will give us more time to come up with simpler text for the titles.